pkg/store/proxy.go

```diff
@@ -371,6 +371,7 @@ func (s *ProxyStore) Series(originalRequest *storepb.SeriesRequest, srv storepb.
 		}
 
 		if err := srv.Send(resp); err != nil {
+			level.Error(reqLogger).Log("msg", errors.Wrap(err, "send series response").Error())
```

Comment:
If we are adding a log statement, better to be more explicit about what is happening. 

```suggestion
			level.Error(reqLogger).Log("msg", "failed to stream response", "error", err)
```